### PR TITLE
Move to GitHub Actions

### PR DESCRIPTION
**Pull request by juanpicado**
Ref https://github.com/Level/community/issues/99

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [6, 8, 10]
+    name: Node ${{ matrix.node }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use node ${{ matrix.node }}
+        uses: actions/setup-node@v1
```
Comment (vweevers) [inline .github/workflows/test.yml:15]:
```suggestion
        uses: actions/setup-node@v2
```

**Comment by juanpicado** [comment]
Unfortunatelly does not run on Node6 and 10, only in 8.

**Comment by vweevers** [comment]
Fixed in https://github.com/Level/mem/commit/19e15a476962b243696968c48bed3fc533c39ed8

**Comment by juanpicado** [comment]
Ohh great, I thought that was a travis thingy :) thanks !

**Pull request merged**